Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
I don't think we can just overwrite all user configuration without prompting here. This also will still do `etc.install`-like behaviour for bottles. We'll need to do something (maybe with `inreplace`?) in `post_install` instead.